jme3-lwjgl/src/main/java/com/jme3/audio/lwjgl/LwjglALC.java

```diff
@@ -12,10 +12,36 @@
 
     @Override
     public void createALC() {
-        try {
-            AL.create();
-        } catch (LWJGLException ex) {
-            throw new RuntimeException(ex);
+        int numRetriesRemaining = 4;
+        int retryDelayMsec = 100; // start with an 0.1-second delay
+
+        while (true) {
+            try {
+                AL.create();
+                break;
+
+            } catch (LWJGLException exception1) {
+                if (numRetriesRemaining < 1) {
+                    throw new RuntimeException(exception1);
+                }
+
+                // Retry to mitigate JME Issue 1383.
+                --numRetriesRemaining;
+                System.out.printf("Caught an LWJGLException from AL.create(). "
```

Comment:
Can this be printed as a warn by the logger?

Reply:
Thanks for the suggestion. Implemented in b546358